Makefile

```diff
@@ -45,7 +49,7 @@ GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
 
 .DEFAULT_GOAL := default
-default: embedded-cluster-linux-amd64
+default: build-ttl.sh
```

Comment:
I think this will be compatible with kots e2e tests

https://github.com/replicatedhq/kots-regression-automation/blob/b429581838a54834609e2b95a77857d5759b2eb4/cluster/templates/control-plane-init-9.sh.tpl#L36